{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth - invalid-api-key", "status": "passed", "description": "Негативная проверка с передачей неверного ключа ", "steps": [{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth на код 401", "status": "passed", "start": 1753848463339, "stop": 1753848464550}, {"name": "Проверка текста message. Ожидали текст 'Вы не авторизованы.                 Пожалуйста, выполните вход и повторите попытку.' Получили 'Вы не авторизованы. Пожалуйста, выполните вход и повторите попытку.'", "status": "passed", "start": 1753848464805, "stop": 1753848464805}], "start": 1753848463339, "stop": 1753848464805, "uuid": "cfeb7fd1-acf3-4220-bed1-4fee9acf4e33", "historyId": "f6aad7747424500f0f55fb65296e0431", "testCaseId": "f6aad7747424500f0f55fb65296e0431", "fullName": "tests.test_api.TestApiGet#test_auth_wrong_key", "labels": [{"name": "tag", "value": "apitest"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApiGet"}, {"name": "host", "value": "Robotek3000"}, {"name": "thread", "value": "12500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}